src/app/utils.py

```diff
@@ -30,6 +30,11 @@
 
 from app.models import User, UserID
 
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
```

Comment:
Should be load all variable in src/secret.py and then import it here
`from src.secret import getRedisHost()` ?

Reply:
@rtgdk Done!